src/OpenDrive2Roadway.jl

```diff
@@ -0,0 +1,9 @@
+module OpenDrive2Roadway
```

Comment:
I think we should call it something like OpenDriveToRoadway rather than with the numerical 2 (for readability, numerals are usually for versions)

Reply:
Sounds good, I'll update this.